dev_guide/jobs.adoc

```diff
@@ -76,9 +78,10 @@ spec:
 A job can be scaled up or down by changing the `*parallelism*` parameter
```

Comment:
The first two sentences feel redundant. How about:
```
A job can be scaled up or down by changing the `*parallelism*` parameter
accordingly ie. setting the maximum number of pods that can run in 
parallel. This can be done via the `oc scale` command with the `--replicas`
option.

The following command uses the example job above, and scales it to three:
 ```

Or something better worded that merges them. Darn, I am not a writer:)